tests/ui/abi/compatibility.rs

```diff
@@ -1,5 +1,5 @@
 //@ check-pass
-//@ revisions: host
```

Comment:
I think we can still have the host revision, we just don't want all the `cfg(host)` stuff any more and instead use the minicore approach for all revisions.

Reply:
I retained the host revision.